models/OrganisationRestApiModel.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const HmpoModel = require('hmpo-model');
+const { iamApi } = require('../config');
+const baseUrl = `${iamApi.protocol}://${iamApi.host}:${iamApi.port}`;
+
+class OrganisationRestApiModel extends HmpoModel {
+
+  requestConfig(config, args) {
+    const retConfig = Object.assign({}, config);
```

Comment:
Should this be "reqConfig", rather than ret?

Reply:
Can potentially be changed but the function this is overriding uses retConfig in the framework:

https://github.com/HMPO/hmpo-model/blob/master/lib/remote-model.js#L70

So I haven't changed but can do if required

Reply:
Nah. From what you've said, it sounds like this if the convention and that trumps a minor rename IMO